content/api/customer-alerts.apib

```diff
@@ -0,0 +1,314 @@
+FORMAT: 1A
+title: Customer-Alerts
+description: Configure real-time alerts for your SparkPost account.
+
+# Group Alerts
+Alerts can be used to get notifications regarding various activities and metrics on your SparkPost account.
+
+Alerts can be configured to send notifications via email to up to 10 email addresses, and supports the following metrics:
+* Real-Time
+    + Monthly Sending limit
+* Signals
+    + Health Score
+
+Alerts are configured to be account-wide for Monthly Sending Limit.
+
+## Metric Limitations
+
+For each metric group, some configuration options simply don't make sense. The following metrics have restrictions:
+
+* Real-Time
+    + interval - Must be '15 Minutes'
+    + filters - filter_type can be 'Sending Domain', 'Sending IP', or 'Mailbox Provider'
+    + threshold_evaluator - source must be 'Raw'
+* Signals
+    + interval - Must be '24 Hours'
+    + filters - filter_type can be 'Sending Domain', 'IP Pool', or 'Mailbox Provider'. Only one filter type can be selected,
+        though multiple values are allowed.
+
+* Monthly Sending Limit (Real-Time)
+  + filters - must be an empty array. No filters may be applied to this metric.
+
+## Filter Object
+
++ Data Structure: Attributes
+  + filter_type (enum) - Type of filter to apply to the alert.
+    + `"Sending Domain", "Mailbox Provider", "Sending IP", "IP Pool"`
+  + `filter_values` (array) - Set of all filter values associated with the given filter_type.
+
++ Sample
+
+  ```
+  {
+    "filter_type": "Sending Domain",
+    "filter_values": [
+      "sparkpost.com",
+      "test.com"
+    ]
+  }
+  ```
+
+## Alert Object
+
++ Data Structure: Attributes
+    + id (number) - Unique identifier of the alert.
+    + name (string) - Display name of the alert.
+    + sid (number) - Subaccount filter applied to the alert.
+    + any_sid (boolean) - Flag indicating all subaccounts should be evaluated for this alert.
+    + metric (enum) - Metric that the alert is configured to track.
+        + `"Monthly Sending Limit", "Health Score"`
+    + interval (enum) - Interval at which the alert is evaluated.
+        + `"24 Hours", "15 Minutes"`
+    + evaluation_enabled (boolean) - Flag indicating whether the alert should be evaluated.
```

Comment:
Can this parameter be simpler just named `enabled`? (No strong opinion just raising the question)

Reply:
Will rename to muted.